src/sparseml/pytorch/yolov8/trainers.py

```diff
@@ -268,7 +268,7 @@ def _setup_train(self, rank, world_size):
             self.steps_per_epoch = len(self.train_loader)  # / self.accumulate
 
             self.scaler = self.manager.modify(
-                self.model,
+                self.model.module if hasattr(self.model, "module") else self.model,
```

Comment:
is this beacuse DataParallel wraps the model in module?

Reply:
interesting, there was at least a patch at some point to make this the default inside of modify, maybe it was a different codepath or the patch wasn't merged in due to testing that never occurred